components/slices/Stat.vue

```diff
@@ -19,14 +21,29 @@ export default {
       default: 0,
     },
   },
+  data() {
+    return {
+      value: [],
+      label: [],
```

Comment:
```suggestion
      values: [],
      labels: [],
```

Suggestion: Mettre au pluriel vu qu'il s'agit d'un tableau.